# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1445391114 14400
#      Tue Oct 20 21:31:54 2015 -0400
# Branch stable
# Node ID 375cea8d41354cc974eabb913946fe815c592671
# Parent  ba7aae4ffe9824f2af04b91cf2be50dfa8c5b710
build: Add check for sys/stropts.h system header file

* configure.ac: Add check for sys/stropts.h system header file.  Fixes build
failure in libgui on Solaris.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2155,17 +2155,17 @@ AC_HEADER_SYS_WAIT
 
 ## C headers
 
 dnl Use multiple AC_CHECKs to avoid line continuations '\' in list
 AC_CHECK_HEADERS([curses.h direct.h dlfcn.h floatingpoint.h fpu_control.h])
 AC_CHECK_HEADERS([grp.h ieeefp.h inttypes.h locale.h memory.h ncurses.h])
 AC_CHECK_HEADERS([poll.h pthread.h pwd.h sunmath.h sys/ioctl.h])
 AC_CHECK_HEADERS([sys/param.h sys/poll.h sys/resource.h])
-AC_CHECK_HEADERS([sys/select.h termcap.h])
+AC_CHECK_HEADERS([sys/select.h sys/stropts.h termcap.h])
 
 ## C++ headers
 
 AC_LANG_PUSH(C++)
 
 AC_CHECK_HEADERS([sstream])
 OCTAVE_UNORDERED_MAP_HEADERS
 
